Core/Src/segment.c

```diff
@@ -5,8 +5,55 @@
 #include <stdlib.h>
 #include <string.h>
 
+// TEMPORARY
+
+#include "common.h"
+#include "adBms6830CmdList.h"
+#include "adBms6830GenericType.h"
+#include "serialPrintResult.h"
+#include "adBms6830ParseCreate.h"
+#include "mcuWrapper.h"
+
+#define ALL_GPIOS_ARE_INPUTS 0x3FF
+
+typedef enum {
+	DISCHARGE_ENABLED = 0,
+	MUTE_ACTIVATED_DISCHARGE_DISABLED = 1
+} MUTE_ST;
+
+#define TOTAL_IC 1
+cell_asic IC[TOTAL_IC];
+
+RD REDUNDANT_MEASUREMENT = RD_OFF;
+CH AUX_CH_TO_CONVERT = AUX_ALL;
```

Comment:
we need to setup which AUX channel is read by which multiplexer.  The mtg notes I think say the main multiplexer reads internal register and the AUX is dedicated to the GPIOs.

Reply:
These variables won't make it into the final draft. this was copied over from adbms source.

Reply:
They'll be gone once this code is tested.